### ISSUE TITLE
@Column attributes to not insert, select or update not working

### ISSUE THREAD

**Issue by roberto-souza** [kibae/typeorm-auditing#5]
Hello,

I use some attributes, such as userId, in my main class with `select`, `insert` and `update` as `false`, just to get value and save in the audit in other fields.

Do you have some suggestions or fix the problem?

For example:

My main entity:
```ts
  @Column({ select: false, insert: false, update: false })
  userId?: string;
```

My audit entity, extends from main entity:
```ts
  @BeforeInsert()
  storeUserId(): void {
    if (this.userId) this._userId = this.userId;
  }

  @Column()
  _userId?: string;
```

**Comment by kibae** [comment]
Hi @roberto-souza,
I modified it to copy all but the column options below.
I have published version 0.5.3, so please check again. 🙏
```
generated,
generatedType,
generatedIdentity,
asExpression,
foreignKeyConstraintName,
primaryKeyConstraintName,
onUpdate,
unique,
```

**Comment by kibae** [comment]
@roberto-souza Sorry, There was a problem, so I published 0.5.4 in a hurry. Please check with version 0.5.4.

**Comment by roberto-souza** [comment]
Now working perfectly! Thank you @kibae!
